x/marker/module.go

```diff
@@ -177,8 +177,8 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 
 // ProposalContents returns all the marker content functions used to
 // simulate marker governance proposals.
-func (am AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalContent {
-	return simulation.ProposalContents(am.keeper)
+func (am AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalMsg {
+	return []simtypes.WeightedProposalMsg{}
 }
```

Comment:
This function can be deleted. It's not longer required by `AppModuleSimulation`. It is part of a singular interface, though, so it'll get used if it exists. But it'd have to have the old signature. There's a new version that returns `[]WeightedProposalMsg`, but that's got a slightly different name. But for now, we should just delete this one.